### ISSUE TITLE
default-features=false -> bevy_ecs fails to resolve

### ISSUE THREAD

**Issue by mosure** [bevyengine/bevy#7963]
## Bevy version

0.10.0

## system information

Windows 11

```
❯ cargo --version
cargo 1.67.1 (8ecd4f20a 2023-01-10)
```

### cargo.toml
```
[dependencies]
bevy = { version = "0.10.0", default-features=false, features = [
  "animation",
  "bevy_asset",
  "bevy_audio",
  "bevy_gilrs",
  "bevy_scene",
  "bevy_winit",
  "bevy_core_pipeline",
  "bevy_pbr",
  "bevy_gltf",
  "bevy_render",
  "bevy_sprite",
  "bevy_text",
  "bevy_ui",
  "png",
  "hdr",
  "ktx2",
  "zstd",
  "vorbis",
  "x11",
  "filesystem_watcher",
  "android_shared_stdcxx",
  "tonemapping_luts",
]}
```

- `cargo build`

## What went wrong

```
error[E0433]: failed to resolve: use of undeclared crate or module `bevy_ecs`
  --> src\lib.rs:39:10
   |
39 | #[derive(Component)]
   |          ^^^^^^^^^ use of undeclared crate or module `bevy_ecs`
```

## Additional information

- when `default-features=false`, even with the set of default features provided, `bevy_ecs` is not being included
- possibly related to: https://github.com/bevyengine/bevy/issues/7854

**Comment by rambip** [comment]
Is it solved now, and if so which version of bevy do I have to use ?
I get exactly the same error on 0.12

**Comment by BorisBoutillier** [comment]
> Is it solved now, and if so which version of bevy do I have to use ? I get exactly the same error on 0.12

Can you check if this is fixed with the bevy 0.13 version that has been released this week-end ? 
If not you should reopen this issue.
